### PR TITLE
component and routes added for single review

### PR DESCRIPTION
**Pull request by JLaw96**

### CODE REVIEW

File: src/components/navigation-bar/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Link } from "react-router-dom";
+const Nav = () => {
+  return (
+    <ul>
+      <button>Categories</button>
+      <button>
```
Comment (JVal-NC) [inline src/components/navigation-bar/Nav.jsx:6]: Could these be different html tags for something more typical used in UI?

File: src/components/reviews/Single-Review.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react";
```
Comment (JVal-NC) [inline src/components/reviews/Single-Review.jsx:2]: need a link to this page as I mentioned in our call, cant rely on the url to navigate to the component. Take a look at the react router notes if needed

File: src/components/reviews/Single-Review.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getReviewById(review_id).then((reviewData) => {
+      setReview(reviewData);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+  console.log(review, "single review");
```
Comment (JVal-NC) [inline src/components/reviews/Single-Review.jsx]: Remove console logs please :)

File: src/components/reviews/Single-Review.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  if (isLoading) {
+    return <p>Loading...</p>;
+  } else {
+    return (
+      <main>
+        <h1>Single Review</h1>
+        <ul>
```
Comment (JVal-NC) [inline src/components/reviews/Single-Review.jsx:24]: Can we style this just so the bullet point isn't visible

**Review by JVal-NC — COMMENTED** [review]
Hi Johnny, good work. Main thing to change is the navigation, refer to react router notes if needed.

**Review by pippaamy — APPROVED** [review]
Good work, I would also show the body on the single review page so they can see more about the article!

**Pull request merged**